OpenRA.Game/Network/OrderManager.cs

```diff
@@ -21,9 +21,23 @@ namespace OpenRA.Network
 {
 	public sealed class OrderManager : IDisposable
 	{
+		const double CatchupFactor = 0.1;
```

Comment:
Can we please have comments explaining what these numbers mean? (e.g. SimLagLimit seems to be in milliseconds, so this is the maximum jank that is allowed before... something?)